pkg/term/term_windows.go

```diff
@@ -86,10 +133,17 @@ func SaveState(fd uintptr) (*State, error) {
 // -- See http://msdn.microsoft.com/en-us/library/windows/desktop/ms683462(v=vs.85).aspx
```

Comment:
BTW: I just checked and not all are available as HTTPS (e.g., http://invisible-island.net/xterm/ctlseqs/ctlseqs.html). I'll update those that are.